### ISSUE TITLE
Common header file (suggestion)

### ISSUE THREAD

**Issue by accelerated** [mfontanini/cppkafka#58]
Would be nice to have a single header file which includes everything one might need when using cppkafka instead of including several header files. This common header should also include the util classes. It would be similar to rdkafka.h for instance. It's always easier to just include a single file and not have to worry about other dependencies. 

suggestion: `cppkafka/cppkafka.h`

Thoughts?

**Comment by mfontanini** [comment]
Sounds good! Ideally this would be generated automatically by CMake as otherwise it's hard to maintain manually otherwise. I don't think that would be too hard to implement but I'm not sure. Otherwise a manually generated header would work for now.

**Comment by accelerated** [comment]
Didn't think about the auto gen header but that sounds even better. With cmake you can GLOB files. I will look into that and submit a PR.

**Comment by mfontanini** [comment]
Yeah, that should work. Also probably sort them (I imagine CMake has a sort function) so it's consistent across platforms and new files don't alter the entire list of headers' order.
